src/main/resources/db/migration/V20220322_56__HMAN-56-AlterLinkTypeEnum.sql

```diff
@@ -0,0 +1,137 @@
+DROP TYPE IF EXISTS public.link_type_enum CASCADE;
```

Comment:
This comment applies to the whole class. Most of these tables were setup as part of HMAN-101, and have been tested by that. Can we reduce the file down to only the changes required otherwise HMAN-101 will require a full re test

Reply:
Can we talk about this? Brill if there is an easier way just to change the enum def.